yt/utilities/lib/fnv_hash.pxd

```diff
@@ -0,0 +1,21 @@
+"""
+Definitions for fnv_hash
+
+
+
+
+"""
+
+#-----------------------------------------------------------------------------
+# Copyright (c) 2017, yt Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+
+import numpy as np
+cimport numpy as np
+
+cdef np.int64_t _fnv_hash(unsigned char[:] octets)
```

Comment:
Minor style nit, maybe call this `c_fnv_hash`? I think that there's a `fnv_hash` exposed in python and `_fnv_hash` exposed in cython is a bit confusing. Took me a minute or two to figure out anyway.